src/main/java/com/harrison/sort/MergeSort.java

```diff
@@ -0,0 +1,9 @@
+package com.harrison.sort;
+
+public class MergeSort {
+
+    public void sort(Integer[] input) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ec2-54-187-222-128.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AS1186)